lm-core/src/main/scala/sbt/librarymanagement/LibraryManagementSyntax.scala

```diff
@@ -35,6 +37,9 @@ trait LibraryManagementSyntax
   // java.lang.System is more important, so don't alias this one
   //  final val System = C.System
   final val Optional = C.Optional
+
+  given Conversion[(String, URI), License] with
+    def apply(x: (String, URI)): License = License(x._1, x._2)
```

Comment:
can/should this be `inline def` ?

Reply:
inline often triggers a bug in macro, but I can see if it would work here.

Reply:
Addressed in 0fd2a356b50c4d115ec615cc95a62a5974abe5d2